apps/admin/.env.cloud-run.example

```
@@ -4,16 +4,16 @@
+F3_GOOGLE_API_KEY=AIzaSy...
 OAUTH_CLIENT_SECRET=...
 
 # Per-environment plain env vars
 AUTH_PROVIDER_URL=https://auth.f3nation.com
 F3_API_BASE_URL=https://api.f3nation.com/v1
-NEXT_PUBLIC_ADMIN_URL=https://admin.f3nation.com
-NEXT_PUBLIC_API_URL=https://api.f3nation.com
-NEXT_PUBLIC_MAP_URL=https://map.f3nation.com
-NEXT_PUBLIC_CHANNEL=prod
+F3_ADMIN_BASE_URL=https://admin.f3nation.com
+F3_API_BASE_URL=https://api.f3nation.com/v1
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The F3_API_BASE_URL key is duplicated

(DuplicatedKey)